src/com/dotcms/rest/api/v1/sites/ruleengine/rules/RuleTransform.java

```diff
@@ -75,33 +77,50 @@ public Rule applyRestToApp(RestRule rest, Rule rule, User user) {
         return groups;
     }
 
-    public RestRule appToRest(Rule app) {
-        return toRest.apply(app);
-    }
+    public RestRule appToRest(Rule app, User user) {
+        RestRule restRule = toRest.apply(app);
```

Comment:
We need to get out of the habit of returning null when bad things happen. It's a SUPER bad practice. ANd it leads to exactly this kind of defensive null checking that clutters up code is distracting.